### PR TITLE
Rtd test

### PR DESCRIPTION
**Pull request by ilfreddy**

### CODE REVIEW

**Comment by stigrj** [comment]
Wow, that's a lot of commits for +8/-6 line changes :stuck_out_tongue:

File: docs/index.rst [hunk 1]
```diff
@@ -46,3 +46,4 @@ The code can be found on `GitHub <https://github.com/MRChemSoft/mrcpp>`_.
 
    programmers_manual/overview
    programmers_manual/clang_tidy.rst
+   programmers_manual/manual
```
Comment (stigrj) [inline docs/index.rst:49]: This file seems to be missing

**Comment by ilfreddy** [comment]
> Wow, that's a lot of commits for +8/-6 line changes 😛

Not sure what happened. I rebased it on top of master and this is the result...

**Comment by stigrj** [comment]
There are probably many squash merges in the history. I will make another with this one, so you are advised to delete your `rtd-test` branch and start fresh from `master` for the next :slightly_smiling_face:

**Review by stigrj — APPROVED** [review]

**Pull request merged**